packages/components/src/navigator/navigator-provider/component.tsx

```diff
@@ -82,25 +83,32 @@ function goTo(
 		return { currentLocation, focusSelectors };
```

Comment:
If I'm at initial location `/foo` and call `goTo( '/foo' )`, it will reset the `isInitial` flag and that's all. Isn't it going to fire an animation? Like the same UI sliding from right "into itself"?

By the way, all the `?.path` optional chainings are not needed, `.path` is enough.

Reply:
> If I'm at initial location /foo and call goTo( '/foo' ), it will reset the isInitial flag and that's all. Isn't it going to fire an animation? Like the same UI sliding from right "into itself"?

I don't think "to same screen" navigations have ever been supported — for the animation to work, we'd need to create a clone of the current screen to transition to.

> By the way, all the ?.path optional chainings are not needed, .path is enough.

Good point, I pushed a commit removing those optional chainings